Grimoire-mc1.12.2/src/main/java/io/github/crucible/grimoire/mc1_12_2/api/integration/eventhelper/IEventHelperIntegration.java

```diff
@@ -23,22 +23,22 @@
 
 public interface IEventHelperIntegration extends IModIntegration {
 
-    public boolean canBreak(@Nonnull EntityPlayer player, @Nonnull BlockPos pos);
+    boolean canBreak(@Nonnull EntityPlayer player, @Nonnull BlockPos pos);
```

Comment:
Could you please not strip away `public` modifier from interface/annotation methods? I know they aren't necessary, but I prefer them being in place for aesthetical reasons.